docs/source/index.rst

```diff
@@ -104,9 +104,10 @@ Documentation
    :maxdepth: 1
    :caption: Running Jobs
 
+   examples/spot-jobs
```

Comment:
For the `gpu-jupyter` above, maybe we can add another section for how to connect vscode to an existing cluster.

Reply:
Will leave this for the future!